recipes/gensim/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "gensim" %}
+{% set version = "0.13.2" %}
+{% set sha256 = "3671d2f35d71e2bb811f5e80b10092f3e4497e52e09bf7a29ea4d0b681e14c9d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy >=1.3
+    - pyro4 >=4.27
```

Comment:
Then would recommend dropping everything except `python` and `setuptools` from `build`.